easybuild/easyconfigs/s/sinto/sinto-0.7.4-foss-2021a.eb

```
@@ -0,0 +1,37 @@
+easyblock = 'PythonPackage'
+
+name = 'sinto'
+
+version = '0.7.4'
+
+homepage = 'https://timoast.github.io/sinto/index.html'
+
+description = """Sinto is a toolkit for processing aligned single-cell data. Sinto includes functions to:
+
+    Subset reads from a BAM file by cell barcode
+
+    Create a scATAC-seq fragments file from a BAM file
+
+    Add read tags to a BAM file according to cell barcode information
+"""
```

I think I would either just go for:
```suggestion
description = """Sinto is a toolkit for processing aligned single-cell data."""
```
or I'd add the entire list of functions that they also include on their site?